tools/check_path_modified.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script is designed to be run inside a GitHub Action job. It
+# checks whether the current commit has modified anything under the
+# given CHECK_PATH. If so, it sets the given ENV_VAR to "true".
+
+if [ "$#" -ne 2 ]; then
+    echo "USAGE: $0 check_path env_var" >& 2
+    echo "Example: $0 images/plbase plbase_modified" >& 2
+    exit 1
+fi
+
+CHECK_PATH=$1
+ENV_VAR=$2
+
+if git diff --exit-code HEAD~1..HEAD -- ${CHECK_PATH}; then
```

Comment:
The gaps in my shell script knowledge are showing 😂 I thought this was backwards at first, but it turns out shells treat `0` as true. The script is fine, this was just a TIL for me!

Reply:
Yeah, it's the world's most confusing thing!

Reply:
Feel free to ping me on bash nonsense. (I've had to work with way too much bash since last year due to some poor choices.)